### PR TITLE
OLS-419: Check content type in e2e tests

### PR DESCRIPTION
**Pull request by tisnik**
## Description

Check content type in e2e tests

## Type of change

- [ ] Refactor
- [ ] New feature
- [ ] Bug fix
- [ ] CVE fix
- [ ] Optimization
- [ ] Documentation Update
- [ ] Configuration Update
- [ ] Bump-up dependent library
- [x] e2e tests

## Related Tickets & Documents

- Related Issue #[OLS-419](https://issues.redhat.com//browse/OLS-419)
- Closes #

## Checklist before requesting a review

- [x] I have performed a self-review of my code.
- [x] PR has passed all pre-merge test jobs.
- [ ] If it is a core feature, I have added thorough tests.

### CODE REVIEW

File: tests/e2e/test_api.py [hunk 1]
```diff
@@ -91,11 +91,17 @@ def get_eval_question_answer(qna_pair, qna_id, scenario="without_rag"):
     return eval_query, eval_answer
 
 
+def check_content_type(response, content_type):
+    """Check if response content-type is set to defined value."""
+    return response.headers["content-type"].startswith(content_type)
```
Comment (onmete) [inline tests/e2e/test_api.py:96]: return -> assert?

Comment (tisnik) [reply tests/e2e/test_api.py:96]: ah indeed (I had different logic implemented, then forget to change, good eyes)

**Pull request merged**